lua/bufferline/highlight.lua

```diff
@@ -10,6 +10,7 @@ hl.set_default_link('BufferAlternateERROR', 'BufferDefaultAlternateERROR')
 hl.set_default_link('BufferAlternateHINT', 'BufferDefaultAlternateHINT')
 hl.set_default_link('BufferAlternateIcon', 'BufferDefaultAlternateIcon')
 hl.set_default_link('BufferAlternateIndex', 'BufferDefaultAlternateIndex')
+hl.set_default_link('BufferAlternateNumber', 'BufferDefaultAlternateNumber')
```

Comment:
This allows a user to do `require'bufferline`.setup {buffer_index = true, buffer_number = true}` and be able to tell the difference between the two.